.chloggen/change-tsprecision-adx-receiver.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
This CHANGELOG file does not match this PR. I believe the solution here is to just not include a CHANGELOG, since this is just a codeowners update we don't need one.

Reply:
Indeed, please remove changelog. Thanks @crobert-1